achyut/src/main/java/com/lftechnology/programs/SalutationProgram.java

```diff
@@ -0,0 +1,49 @@
+package com.lftechnology.programs;
+
+import java.util.Scanner;
+
+public class SalutationProgram {
+
+    private Scanner fullNameWithSalutationInput;
+    private String fullNameWithSalutation;
+    private String salutation;
+
+    public SalutationProgram() {
+
+    }
+
+    public void run() {
+        this.initiateRun();
+        this.askForInput();
+        if (this.fullNameWithSalutation.length() <= 0) {
+            System.out.println("Input can't be empty! Enter again");
+            this.askForInput();
+        }
+        this.splitInput();
+        System.out.println("Salutation = " + this.salutation);
+        System.out.println("FullName = " + this.fullNameWithSalutation);
+        this.endRun();
+    }
+
+    public void initiateRun() {
+        System.out.println("\n");
+        System.out.println("Executing Salutation");
+        System.out.println("Please Enter Your Full Name with Salutation (e.g, Mr. John Doe)\n\n");
+    }
+
+    public void endRun() {
+        System.out.println("-----------------------------------------");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)